README.md

```diff
@@ -1,17 +1,167 @@
-# Sightglass
-
-A benchmark suite and tool to compare different implementations of the same primitives. It contains
-several parts:
```

Comment:
I think removing this documentation should mean that we accelerate the removal of the legacy infrastructure. The steps for this are listed in https://github.com/bytecodealliance/sightglass/issues/95. I'm fine with this, but let me cc @jlb6740.